elasticsearch/src/main/java/zipkin/dependencies/elasticsearch/ElasticsearchDependenciesJob.java

```diff
@@ -56,6 +56,15 @@ public static Builder builder() {
       sparkProperties.put("es.nodes.wan.only", getEnv("ES_NODES_WAN_ONLY", "false"));
       // NOTE: unlike zipkin, this uses the http port
       sparkProperties.put("es.nodes", getEnv("ES_HOSTS", "127.0.0.1"));
+      sparkProperties.put("es.net.ssl", getEnv("ES_SSL", "false"));
```

Comment:
for testability, you could extract this into a static method that takes the `Map<String, String>` from `System.getenv()`